WakaTime.py

```diff
@@ -194,11 +194,32 @@ def read(self):
                     if key:
                         self._key = key
                         return self._key
+        
+            apiKeyFromVault = self.__readFromVaultCmd()
+            if apiKeyFromVault:
+                self._key = apiKeyFromVault
+                return self._key
         except:
             pass
 
         return self._key
 
+    def __readFromVaultCmd(self):
```

Comment:
Let's name this `def api_key_from_vault_cmd(self):`. You can add a single underscore if you want to convey it's a private method, but since we don't have any public api here it's not necessary.